public/index.html

```diff
@@ -1,29 +1,49 @@
-<!doctype html>
-<html lang="en" dir="ltr">
-
-<head>
-    <title>Dashboards app</title>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-
-    <%= htmlWebpackPlugin.options.vendorScripts %>
-
-
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+        <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1, shrink-to-fit=no"
+        />
+        <meta name="theme-color" content="#000000" />
         <!--
-  Notice the use of %PUBLIC_URL% in the tag above.
-  It will be replaced with the URL of the `public` folder during the build.
-  Only files inside the `public` folder can be referenced from the HTML.
-
-  Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-  work correctly both with client-side routing and a non-root public URL.
-  Learn how to configure a non-root public URL by running `npm run build`.
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-</head>
+        <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+
+        <% if (process.env.NODE_ENV === 'production') { %>
+        <script src="../dhis-web-core-resource/babel-polyfill/6.20.0/dist/polyfill.min.js"></script>
+        <script src="../dhis-web-core-resource/jquery/3.2.1/dist/jquery.min.js"></script>
+        <script src="../dhis-web-core-resource/jquery-migrate/3.0.1/dist/jquery-migrate.min.js"></script>
+        <script src="../dhis-web-pivot/reporttable.js"></script>
+        <script src="../dhis-web-maps/map.js"></script>
+        <script src="../dhis-web-event-reports/eventreport.js"></script>
+        <script src="../dhis-web-event-visualizer/eventchart.js"></script>
+        <% } %> <% if (process.env.NODE_ENV !== 'production') { %>
+        <script src="%REACT_APP_DHIS2_BASE_URL%/dhis-web-core-resource/babel-polyfill/6.20.0/dist/polyfill.min.js"></script>
+        <script src="%REACT_APP_DHIS2_BASE_URL%/dhis-web-core-resource/jquery/3.2.1/dist/jquery.min.js"></script>
```

Comment:
As an interim solution shouldn't we be able to include the static `jquery` files and reference them locally here?

That would still expose them in the global namespace and doesn't require bundling into each plugin (though we should eventually figure out a better solution for that) while removing the core-resource dependency.

Reply:
Yes.

Reply:
@varl @jenniferarnesen I just pushed a commit with this change.

Reply:
This brings me back to 2008. <3

Reply:
You know would be super cool is to remove the dependency on jQuery (not now, but down the road). Not sure what that would entail.

Reply:
@jenniferarnesen FWIW we use `%PUBLIC_URL%` to reference files relative to the app root, rather than the DHIS2 core root.  We don't want to assume that we're installed at a particular sub-url on the DHIS2 server (though currently we are and are using those URLs to reference the plugins)

Reply:
Yepp.